contracts/ROSCA.sol

```diff
@@ -130,10 +150,47 @@ contract ROSCA {
 
       currentRound++;
       LogStartOfRound(currentRound);
+    } else if (currentRound == membersAddresses.length) {
+        endOfROSCA = true;
     }
   }
 
   /**
+   * try to send the fund to the opt_destination from source account
+   */
+  function sendFund(address source,address opt_destination) internal returns(bool success){
+    uint amountToWithdraw = members[source].pendingWithdrawal  ;  // use a temporary variable to avoid the receiver calling the withdraw function again before
+    if(this.balance < amountToWithdraw) amountToWithdraw = this.balance;
+    members[source].pendingWithdrawal = members[source].pendingWithdrawal - amountToWithdraw;                      // send() complete its task
+    if (!opt_destination.send(amountToWithdraw)) {   // if the send() fails, put the allowance back to its original place
```

Comment:
i think there is a bug here: send() expects amount in wei. 
amountToWithdraw is in finney. Let's switch to using wei everywhere to avoid such mistakes.